src/libs/ajax.js

```diff
@@ -860,8 +860,27 @@ export const User = {
     }
   },
 
+  updateSelf: async (user) => {
+    const url = `${await Config.getApiUrl()}/api/user`;
+    // We should not be updating the user's create date, associated institution, or library cards
+    let filteredUser = flow(
+      cloneDeep,
+      unset('updatedUser.createDate'),
+      unset('updatedUser.institution'),
+      unset('updatedUser.libraryCards')
```

Comment:
I think we're mixing concepts here. The object passed in shouldn't actually be a full `user` object, but instead a custom object with a limited set of fields as defined in the server side API.